prow/oss/terraform/modules/alerts/main.tf

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+locals {
+  playbook_prefix = "https://github.com/google/exposure-notifications-verification-server/blob/main/docs/playbooks/alerts"
+}
+
+resource "google_monitoring_alert_policy" "plank-alert" {
+  project      = var.project
+  display_name = "Heartbeat ProwJobs"
```

Comment:
The display name should be consistent with the alert name, right? So I think we should name this something specific to heartbeat prowjobs rather than `plank-alert`.

Reply:
good point, updated